File: Android/Martinshare/app/src/main/java/mobile/martinshare/com/martinshare/API/protocols/FeedbackProtocol.java
package mobile.martinshare.com.martinshare.API.protocols;

/**
 * Created by Modestas Valauskas on 06.06.2015.
 */
public interface FeedbackProtocol {
    public void startedFeedback();
    public void feedbacksent();
    public void isNotLoggedIn();
    public void noInternet();
    public void unknownError(final String error);
}

File: iOS/Martinshare/Martinshare/FeedbackProtocol.swift
//
//  FeedbackProtocol.swift
//  Martinshare
//
//  Created by Modestas Valauskas on 07.12.15.
//  Copyright © 2015 ModestasV Studios. All rights reserved.
//

import Foundation

protocol FeedbackProtocol {
    
    func startedConnection()
    func notLoggedIn()
    func unknownError()
    func noInternetForSending()
    func sent()
    func error(_ textshow: String)
}
